docs/development/extensions-contrib/compressed-big-decimal.md

```diff
@@ -144,6 +144,31 @@ IngestionSpec syntax:
 	}
 }
 ```
+
+Sql based ingestion sample query:
+```sql
+
+REPLACE INTO "bigdecimal" OVERWRITE ALL
+WITH "ext" AS (
+  SELECT *
+  FROM TABLE(
+    EXTERN(
+      '{"type":"local","baseDir":""/home/user/sales/data/staging/invoice-data","filter":"invoice-001.20201208.txt"}',
+      '{"type":"csv","findColumnsFromHeader":false,"columns":["timestamp","itemName","saleAmount"]}',
+      '[{"name":"timestamp","type":"string"},{"name":"itemName","type":"string"},{"name":"saleAmount","type":"double"}]'
+    )
+  ) 
+)
+SELECT
+  TIME_PARSE(TRIM("timestamp")) AS "__time",
+  "itemName",
+  BIG_SUM("saleAmount") as amount
```

Comment:
I guess descendants of `CompressedBigDecimalSqlAggregatorTestBase` is not run under msq as it would be wierd to load that under the msq module.

I wonder if it would be possible to either run those tests somehow from the `quidem-ut` module - or possibly just excercise some query via an `iq` file from the same module to ensure that it can't be broken too easily